stock_dropshipping_dual_invoice/tests/test_need_two_invoices.py

```diff
@@ -73,14 +73,30 @@ def test_so_on_delivery_need_one_invoice(self):
         }).create({})
         self.assertIs(False, wizard._need_two_invoices())
 
+    def test_so_mto_need_one_invoice(self):
+        self.so_mto.order_policy = 'picking'
+        self.so_mto.action_button_confirm()
+        procurements = self.so_mto.procurement_group_id.procurement_ids
+        procurements.run()
+        po = procurements.filtered('purchase_id')[:1].purchase_id
+        self.assertTrue(po)
+        po.invoice_method = 'picking'
+        po.signal_workflow('purchase_confirm')
+        picking = po.picking_ids[:1]
+        picking.action_done()
+        wizard = self.Wizard.with_context({
+            'active_id': picking.id,
+            'active_ids': [picking.id],
+        }).create({})
+        self.assertFalse(wizard._need_two_invoices())
```

Comment:
(minor) the assertion `assertIs(False)` is stronger because it checks for exactly `False`. On the other hand, `assertFalse` checks for something that is `False`-ish.

I do that since when I made the following mistake:
`self.assertTrue(x.my_method)`
forgetting to put the parenthesis to actually call the method. The above test passes, while `self.assertIs(True, x.my_method)` fails as it should.